include/crm/common/options_internal.h

```diff
@@ -155,11 +155,6 @@ bool pcmk__valid_stonith_watchdog_timeout(const char *value);
 #define PCMK__ENV_TRACE_TAGS                "trace_tags"
 #define PCMK__ENV_VALGRIND_ENABLED          "valgrind_enabled"
 
-/* @COMPAT Drop at 3.0.0; added to debug shutdown issues when Pacemaker is
- * managed by systemd, but no longer useful.
- */
-#define PCMK__ENV_SHUTDOWN_DELAY            "shutdown_delay"
```

Comment:
This one is in doc/sphinx/Pacemaker_Explained/local-options.rst

Reply:
Added to #3515